apps/src/templates/TeacherSectionOption.jsx

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import color from '@cdo/apps/util/color';
+import {sectionForDropdownShape} from './teacherDashboard/shapes';
+
+export default class TeacherSectionOption extends Component {
+  static propTypes = {
+    section: sectionForDropdownShape,
+    onChange: PropTypes.func.isRequired,
+    isChecked: PropTypes.bool,
+    assignedSections: PropTypes.arrayOf(sectionForDropdownShape)
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      updatedAssignedSections: this.props.assignedSections
+    };
+  }
+
+  renderCheckbox = () => {
+    const {isChecked, onChange} = this.props;
+    return (
+      <input
+        type="checkbox"
+        checked={isChecked}
+        onChange={onChange}
+        style={styles.checkbox}
+      />
+    );
+  };
+
+  render() {
+    const {section} = this.props;
+    return (
+      <div>
+        <span>
+          <div style={styles.sectionOptionContainer}>
+            <div>{this.renderCheckbox()}</div>
+            <label style={styles.sectionOptionLabel}>{section.name}</label>
```

Comment:
When you use labels, they should be associated with an input, in this case the checkbox. You can do this either by wrapping the `<input />` in `<label>` tags or use `htmlFor` ([example](https://github.com/code-dot-org/code-dot-org/blob/2330c548b758cf9be7134d6c601f2ecec7f22955/apps/src/templates/teacherDashboard/MultiSelectGroup.jsx#L77)). It's an accessibility feature so that screen readers can associate the input with what it does.

Reply:
Fixed!